examples/advanced/llm_hf/README.md

```diff
@@ -90,35 +91,77 @@ If the correct "global model" is successfully reloaded each round, the training
 
 To run iterative training, we use the following command:
 ``` 
-python3 ./utils/hf_sft_peft_iter.py --output_path ./workspace/dolly_cen_sft_iter --train_mode SFT
-python3 ./utils/hf_sft_peft_iter.py --output_path ./workspace/dolly_cen_peft_iter --train_mode PEFT
+python3 ./utils/hf_sft_peft_iter.py --output_path ./workspace/dolly_cen_iter 
 ```
 
 The SFT curves are shown below, black for single call, blue for iterative. We can see the "zig-zag" pattern in the iterative training loss curve.
 ![sft](./figs/cen_sft.png)
 
-Similar patterns can be observed from the PEFT curves, purple for single call, green for iterative.
-![peft](./figs/cen_peft.png)
+### Adaptation Step 1.5: iterative training with scheduler alignment and multi-gpu support
+The above iterative training script is a basic version, it works fine if using a constant lr_scheduler. However, if using more complicated lr_scheduler like "cosine_with_restarts", the lr will not be aligned with the iterative training process, which can lead to unexpected results.
```

Comment:
"constant learning rate", not "constant lr_scheduler".

Reply:
corrected